File: 2949/A Sociedade do Anel.py
n = int(input())


hob = 0
hu = 0
el = 0
an = 0
ma = 0
for x in range(n):
    a, b = input().split()

    if b == "X":
        hob+=1
    if b == "M":
        ma+=1

    if b == "H":
        hu+=1
    if b == "E":
        el+=1
    if b == "A":
        an+=1

print("%s Hobbit(s)"%hob)
print("%s Humano(s)"%hu)
print("%s Elfo(s)"%el)
print("%s Anao(s)"%an)
print("%s Mago(s)"%ma)

File: 2949/A Sociedade do Anel.cpp
using namespace std;
#include <iostream>
#include <vector>
#include<bits/stdc++.h>
#include <algorithm>


int main(){

    int t;
    cin >> t;
    string nome, tipo;
    int hob = 0, hu = 0, ma = 0, el = 0, an = 0;
    for(int i = 0; i < t; i++){
         cin >> nome >> tipo;
         
         if (tipo == "A")
            an++;
         if (tipo == "E")
            el++;
         if (tipo == "H")
            hu++;
         if (tipo == "M")
            ma++;
         if (tipo == "X")
            hob++;
         
     }
    cout << hob <<" Hobbit(s)"<< endl;
    cout << hu <<" Humano(s)"<< endl;
    cout << el <<" Elfo(s)"<< endl;
    cout << an <<" Anao(s)"<< endl;
    cout << ma <<" Mago(s)"<< endl;
}
